### PR TITLE
#166790019  Comments Should Be Able To Track Edit History 

### PR DESCRIPTION
**Pull request by kodek-sleuth**
#### What does this PR do?
Should track comment edit history
#### Description of Task to be completed
Setup model and controller to persist data and validation
#### How should this be manually tested?
1. Run the following commands in the terminal
```
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkoutft-track-comment-history-166790019
npm install
npm start
```
2. Create a .env file and add the following snippets
```
DATABASE_URL=postgres://username:password@localhost:5432/authorshaven
DATABASE_=postgres://username:password@localhost:5432/testing-ah
SECRET=whatever you want it to be
```
3. Install `sequelize` globally
```
sudo npm install -g sequelize
```
4.  Create a database using `sequelize db:create`
5. Install and setup Postman to consume CRUD endpoints
6. Run the following requests in Postman using `localhost:3000`
## Endpoints
|  Method  |  Endpoint  |  Task  |
|  --- |  --- |  ---  |
|  `PATCH`  |  `/api/:commentId/history`  |  `track history`  |
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166790019](https://www.pivotaltracker.com/story/show/166790019)
#### Screenshots (if appropriate)
Screen Shot 2019-08-14 at 11 40 13

#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Review by niyoceles — DISMISSED** [review]

File: helpers/sqlRawQueries.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const getArticleAndRatings = articleSlug => `select articles.title, articles.body, articles.image, ratings.rating,
```
Comment (elemanhillary-zz) [inline helpers/sqlRawQueries.js]: we talked about these queries. kindly eliminate them

**Review by elemanhillary-zz — CHANGES_REQUESTED** [review]

**Review by elemanhillary-zz — CHANGES_REQUESTED** [review]
resolve merge conflicts

**Review by IsaiahRn — DISMISSED** [review]

**Review by niyoceles — DISMISSED** [review]

**Review by Hervera — DISMISSED** [review]
Ready to be merged

**Review by elemanhillary-zz — CHANGES_REQUESTED** [review]
resolve merge conflicts

**Review by niyoceles — DISMISSED** [review]

**Review by niyoceles — DISMISSED** [review]

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by IsaiahRn — DISMISSED** [review]

**Review by niyoceles — APPROVED** [review]
ready to be merged

**Review by elemanhillary-zz — CHANGES_REQUESTED** [review]
coverage decreased kindly fix it

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by niyoceles — DISMISSED** [review]

**Review by IsaiahRn — DISMISSED** [review]

**Review by IsaiahRn — APPROVED** [review]

**Review by niyoceles — DISMISSED** [review]

**Review by Hervera — DISMISSED** [review]
Ready to be merged

**Review by IsaiahRn — DISMISSED** [review]

**Review by Hervera — DISMISSED** [review]

File: tests/report.test.js [hunk 2]
```diff
@@ -126,18 +126,18 @@ describe('Testing report an article ', () => {
       });
   });
 
-  it('Should not allow any user to  get report with data', (done) => {
```
Comment (niyoceles) [inline tests/report.test.js]: Kindly don't remove this test

**Review by niyoceles — CHANGES_REQUESTED** [review]

**Review by niyoceles — APPROVED** [review]

**Review by elemanhillary-zz — APPROVED** [review]
nice work, ready for merge

**Review by Hervera — APPROVED** [review]

**Review by IsaiahRn — APPROVED** [review]

**Pull request merged**